src/pages/partners/affiliate-ib/_faq-data.tsx

```diff
@@ -575,101 +573,102 @@ const IBGeneral = () => (
-                type: 'paragraph-2',
-            }}
-        >
-            {localize('How and when will I receive my Deriv IB commissions?')}
-        </Header>
-        <Header
-            as="p"
-            type="paragraph-1"
-            mt="8px"
-            weight="normal"
-            tabletL={{
-                type: 'paragraph-2',
-            }}
-        >
-            {localize(
-                'Your IB commissions are credited directly into your Deriv MT5 account daily. You can transfer the funds to your Deriv account and withdraw them via your preferred payment method.',
-            )}
-        </Header>
-    </ItemContainer>
-)
-
-const IBReferralTools = () => (
-    <ItemContainer>
-        <Header
-            as="p"
-            type="paragraph-1"
-            mt="16px"
-            tabletL={{
-                type: 'paragraph-2',
-            }}
-        >
-            {localize('Do you offer any referral tools to your IBs?')}
-        </Header>
-        <Header
-            as="p"
-            type="paragraph-1"
-            mt="8px"
-            weight="normal"
-            tabletL={{
-                type: 'paragraph-2',
-            }}
-        >
-            {localize(
-                'Definitely. We’ll provide you with banners, videos, reviews, links, and text ads that you can use to bring new clients to our Deriv MT5 platform.',
-            )}
-        </Header>
-    </ItemContainer>
-)
-
+const IBAccountManagement = () => {
+    const [is_livechat_interactive, LC_API] = useLivechat()
+    return (
+        <ItemContainer>
+            <Header
+                as="p"
+                type="paragraph-1"
+                mt="16px"
+                tabletL={{
+                    type: 'paragraph-2',
+                }}
+            >
+                {localize('How can I sign up as a Deriv IB?')}
+            </Header>
+            <Header
+                as="p"
+                type="paragraph-1"
+                mt="8px"
+                weight="normal"
+                tabletL={{
+                    type: 'paragraph-2',
+                }}
+            >
+                <Localize
+                    translate_text="All you need to do is head to the signup page and fill out the application form. We’ll review your application and get in touch if it’s successful."
+                    components={[
+                        <LocalizedLinkText
+                            to={affiliate_signup_url}
+                            external
+                            type="affiliate_sign_up"
+                            target="_blank"
+                            rel="noopener noreferrer"
+                            key={0}
+                        />,
+                    ]}
+                />
+            </Header>
+            <TextSecondary>
+                <Localize translate_text="If you’re already registered as a Deriv affiliate, " />
+                <LiveChatLinkText text="contact us via live chat." />
```

Comment:
```suggestion
    <LiveChatLinkText text="_t_contact us via live chat._t_" />
```
since you used `text` directly in `LiveChatLinkText`, you have to wrap the text with `_t_`.